### PR TITLE
🚀 run jscs over tests

### PR DESCRIPTION
**Pull request by samccone**
ref:
https://github.com/tastejs/todomvc/commit/07353bcd844b41a2bf214d9c42845b65157a8bf7#commitcomment-13197635
https://github.com/tastejs/todomvc/commit/abee8bdedb4913a6f70b4ad5167fad16ad333def

R: @arthurvr

### CODE REVIEW

File: tests/knownIssues.js [hunk 1]
```diff
@@ -4,7 +4,9 @@ module.exports = [
 	// chapling brunch does not mark items as completed via the completed CSS class
 	'TodoMVC - chaplin-brunch, No Todos, should hide #main and #footer',
 	'TodoMVC - chaplin-brunch, Mark all as completed, should allow me to mark all items as completed',
+	/* jscs:disable */
```
Comment (arthurvr) [inline tests/knownIssues.js]: Is there a reason you're using multiline-`/*` comments here while using `//` for the others?

File: tests/knownIssues.js [hunk 2]
```diff
@@ -33,8 +35,10 @@ module.exports = [
 	// this implementation has numerous edit experience issues
 	'TodoMVC - angularjs_require, Mark all as completed, should allow me to mark all items as completed',
 	'TodoMVC - angularjs_require, Mark all as completed, should allow me to clear the completion state of all items',
+	//jscs:disable
```
Comment (arthurvr) [inline tests/knownIssues.js]: `// jscs:disable`, generally we do that space after a comment. Or do you want to handle JSCS comments as an exception?

**Comment by arthurvr** [comment]
Cool, thanks.

**Comment by arthurvr** [comment]
Yay! Thanks @samccone.

**Comment by samccone** [comment]
np thanks for the :eyes:

**Pull request merged**